src/createStripe.ts

```diff
@@ -61,6 +62,8 @@ export function createStripe(
   Stripe.HttpClient = HttpClient;
   Stripe.HttpClientResponse = HttpClientResponse;
   Stripe.CryptoProvider = CryptoProvider;
+  Stripe.Stripe = Stripe;
```

Comment:
I added this to maintain backwards compatibility in the ESM entrypoint, because these properties existed on the object the user imported. The way we export right now (`export const Stripe = ...` and `export default Stripe`) in the ESM entrypoint no longer introduces those properties through `module.exports`.

Reply:
Let's add a comment because it will look strange for a future reader. And maybe consider removing at least for ESM in the next major

Reply:
Filed an issue to remove these in the next major!